plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergFailureRecoveryTest.java

```diff
@@ -110,4 +111,13 @@ public void testInsertIntoExistingPartition()
                 "INSERT INTO <table> SELECT *, 'partition1' p FROM orders",
                 Optional.of("DROP TABLE <table>"));
     }
+
+    @Override
+    protected Session enableDynamicFiltering(boolean enabled)
+    {
+        Session session = super.enableDynamicFiltering(enabled);
+        return Session.builder(session)
+                .setCatalogSessionProperty(session.getCatalog().orElseThrow(), "dynamic_filtering_wait_timeout", "1h")
```

Comment:
.... after the move, the context of  (https://github.com/trinodb/trino/commit/504c9cfcfd863fb7a76ecef0a748a51ff1992579) is irreversibly lost.

add capture code comment explaining why we're using a large value here

Reply:
@raunaqmorarka can you help with this one?

Reply:
Large value is to make the probe side scan wait until DF is collected. The test itself should have a short timeout in case the probe scan never unblocks.

Reply:
thx. Caputred as code comment.